src/components/textarea/examples/TextareaAutosizeExamples.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import TextareaAutosize from 'react-textarea-autosize';

```

Comment:
I think we should have a boolean in our `TextArea` to have it autosize or not instead of importing the component from another lib.

Reply:
this lib use autosize to handle the resize of the textarea. I think its useless to code the same component. this lib is very small and it add only this component.